packages/block-editor/src/components/iframe/index.js

```diff
@@ -242,10 +242,8 @@ function Iframe( {
 	const isZoomedOut = scale !== 1;
 
 	useEffect( () => {
-		if ( ! isZoomedOut ) {
-			prevContainerWidth.current = containerWidth;
```

Comment:
I wonder what led to this change? As I understand it the value of `prevContainerWidth` should be that of `containerWidth` before Zoom Out is engaged because the scale calculation needs to arrive at a scale which preserves the document width. That calculation wasn’t perfect before but this seems to have thrown it off a good deal when Zoom Out is engaged by way of the Inserter (not through the control added by this PR).

Reply:
Good catch, I made a PR for that here: https://github.com/WordPress/gutenberg/pull/64478